src/Entity.ts

```diff
@@ -57,6 +57,7 @@ export function isPrebuilt(entity: EntityBase) {
 }
 
 export const MAX_ENUM_VALUES = 5
+export const MAX_ENUM_VALUE_LENGTH = 10
```

Comment:
Is this the name of the enum?   Might call it "MAX_ENUM_NAME_LENGTH"

Reply:
It's actually for the value -- eg, if the enum is `productType` and the values are `WINDOWS` and `OFFICE`, this would apply to `WINDOWS` and `OFFICE`.

Reply:
Then mabye "MAX_ENUM_VALUE_NAME_LENGTH"?   I found the original name a bit confusing when paired against the other

Reply:
I'm going to push back on this one.   I think the previous one is pretty clearly related to the count of enum values, so I don't find "length" to be ambiguous.

Reply:
or we could change them both - `MAX_ENUM_VALUE_COUNT` and `MAX_ENUM_VALUE_NAME_LENGTH` ?